README.md

```diff
@@ -18,3 +18,27 @@ However, expect things to break or change without warning.
 
 See here for an example: https://github.com/pymc-devs/pymc4/blob/master/notebooks/radon_hierarchical.ipynb
 See here for the design document: https://github.com/pymc-devs/pymc4/blob/master/notebooks/pymc4_design_guide.ipynb
+
+## Develop
+
+One easy way of developing on PyMC4 is to take advantage of the development containers! 
+Using pre-built development environments allows you to develop on PyMC4 without needing to set up locally.
+
+To use the dev containers, you will need to have Docker and VSCode running locally on your machine, and will need the VSCode Remote extension installed inside VSCode.
```

Comment:
Can you be more specific about the VSCode Remote extension? (Which version, or link to marketplace..)

Reply:
Good point! Let me modify.